modules/node_modules/@ncigdc/containers/FrequentlyMutatedGenesTable.js

```diff
@@ -145,6 +146,11 @@ const FrequentlyMutatedGenesTableComponent = compose(
             ),
           },
           {
+            key: 'annotations',
+            title: 'Annotations',
+            style: { padding: '5px 0 0 0', textAlign: 'center' },
```

Comment:
@shanewilson  yea we can. we had some tables in the beginning that didn't any styling, so the styles were just handled at the uikit level, but things grew out of control